lib/src/Puzzle.dart

```diff
@@ -149,6 +149,10 @@ class Puzzle {
   /// I can only make these comments so interesting and no more :l
   void startStopwatch() => _stopwatch.start();
   void stopStopwatch() => _stopwatch.stop();
+  
+  ///to check stopclock is paused or not
+  bool get isStopClockRunning => _stopwatch.isRunning();
```

Comment:
```suggestion
  bool get isStopwatchRunning => _stopwatch.isRunning();
```

Matching the naming of other `Stopwatch` methods.